### ISSUE TITLE
Immediately open articles from a feed in the browser

### ISSUE THREAD

**Issue by j-f1** [Ranchero-Software/NetNewsWire#2268]
Some feeds I subscribe to don’t work properly even in reader mode, and others link to dynamic content that can not be displayed in an RSS reader. Just like there’s a per-feed setting to open reader view immediately, it would be great to have a similar option to open the article in a Safari View Controller or the default browser (on Mac).

**Comment by rosolko** [comment]
Would like to have an ability for ios application also. For example in feedly it called `Open in website directly`, this case you have only title and one-two lines in an article list, but clicking on it application redirect you to embeded browser view instead of render inside application.

**Comment by rosolko** [comment]
@brentsimmons Hi. Is it even possible to add such ability into applications? Can you consider on this? Thanks!

**Comment by brentsimmons** [comment]
We do now have a global setting for in-app vs. default browser on iOS, but we don’t have a way to specify per feed. We’ll take that as a feature request.

**Comment by brentsimmons** [comment]
To be clear — you mean that upon selecting an article in the timeline (the article list), that article would open in your default web browser?

**Comment by rosolko** [comment]
@brentsimmons Yes, exactly. For example, some feeds don't contain any content if you are trying to read then via application or read mode. Only opening them in a default or in-app browser will work.
